pkg/service/backup/backup.go

```diff
@@ -45,6 +49,16 @@ func makeHostInfo(nodes []scyllaclient.NodeStatusInfo, locations []backupspec.Lo
 			hi[i].RateLimit = dcr[""] // no rate limit is ok, fallback to 0 - no limit
 		}
 		hi[i].Transfers = transfers
+
+		ip, err := netip.ParseAddr(h.Addr)
```

Comment:
I saw that you are assuring that the ip is in form of netip.ParseAddr when calling to makeHostInfo from the service. Why it's duplicated here as well ? 
Other method users are not applying this function ?